rp2040-hal/src/spi.rs

```diff
@@ -53,6 +54,16 @@ impl From<&embedded_hal::spi::Mode> for Mode {
     }
 }
 
+/// SPI frame format
+pub enum FrameFormat {
+    /// Motorola SPI format. See section 4.4.3.9 of RP2040 datasheet.
+    MotorolaSpi(Mode),
```

Comment:
`Mode` is a _newtype_ around `embedded_hal::spi::Mode`.

Can we make `FrameFormat` implement `From<_>` for the embedded_hal's `spi::Mode` and ditch our _newtype_?

```rust
impl From<embedded_hal::spi::Mode> for FrameFormat {
    // …
}

#[cfg(feature = "eh1_0_alpha")]
impl From<eh1_0_alpha::spi::Mode> for FrameFormat {
    // …
}
```

This would make the changes to the doc and examples unnecessary.